library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss

```diff
@@ -588,10 +588,12 @@
 
 		.progress-bar-empty {
 			background-color: transparent;
-			color: var(--sakai-text-color-1);
+			font-weight: 600;
 		}
 
-
+		.active {
+			background-color: var(--progress-bar-bg);
```

Comment:
Overwritted `.active` class from _sakai-add-topic.scss (just in case it's being used elsewhere) to use a standard sakai color.